examples/pennylane_run_qaoa_maxcut.py

```diff
@@ -0,0 +1,287 @@
+# the graph. We also define the graph using
+# the list ``graph``, which contains the tuples of vertices defining
+# each edge in the graph.
+
+n_wires = 4
+graph = [(0, 1), (0, 3), (1, 2), (2, 3)]
+
+# unitary operator U_B with parameter beta
+def U_B(beta):
+    for wire in range(n_wires):
+        qml.RX(2 * beta, wires=wire)
+
+
+# unitary operator U_C with parameter gamma
+def U_C(gamma):
+    for edge in graph:
+        wire1 = edge[0]
+        wire2 = edge[1]
+        qml.CNOT(wires=[wire1, wire2])
+        qml.RZ(gamma, wires=wire2)
+        qml.CNOT(wires=[wire1, wire2])
+
+
+##############################################################################
+# With some foresight, we notice that we will need a way to sample
+# a measurement of multiple qubits in the computational basis, so we define
+# a Hermitian operator to do this. The eigenvalues of the operator are
+# the joint qubit measurement values in integer form.
+
+
+def comp_basis_measurement(wires):
+    n_wires = len(wires)
+    return qml.Hermitian(np.diag(range(2 ** n_wires)), wires=wires)
+
+
+##############################################################################
+# Circuit
+# ~~~~~~~
+# Next, we create a quantum device with 4 qubits.
+
+dev1 = qml.device("default.qubit", wires=n_wires, analytic=True, shots=1)
+
+##############################################################################
+# We also require a quantum node which will apply the operators according to the
+# angle parameters, and return the expectation value of the observable
+# :math:`\sigma_z^{j}\sigma_z^{k}` to be used in each term of the objective function later on. The
+# argument ``edge`` specifies the chosen edge term in the objective function, :math:`(j,k)`.
+# Once optimized, the same quantum node can be used for sampling an approximately optimal bitstring
+# if executed with the ``edge`` keyword set to ``None``. Additionally, we specify the number of layers
+# (repeated applications of :math:`U_BU_C`) using the keyword ``n_layers``.
+
+pauli_z = [[1, 0], [0, -1]]
+pauli_z_2 = np.kron(pauli_z, pauli_z)
+
+
+@qml.qnode(dev1)
+def circuit(params, edge=None, n_layers=1):
+    # apply hadamards to get n qubit |+> state
+    for wire in range(n_wires):
+        qml.Hadamard(wires=wire)
+    # p instances of unitary operators
+    for i in range(n_layers):
+        U_C(params[0][i]) # params[0] is gamma
+        U_B(params[1][i]) # params[1] is beta
+    if edge is None:
+        # measurement phase
+        return qml.sample(comp_basis_measurement(range(n_wires)))
+    # during the optimization phase we are evaluating a term
+    # in the objective using exp val
+    return qml.expval(qml.Hermitian(pauli_z_2, wires=edge))
+
+
+##############################################################################
+# Optimization
+# ~~~~~~~~~~~~
+# Finally, we optimize the objective over the
+# angle parameters :math:`\boldsymbol{\gamma}` (``params[0]``) and :math:`\boldsymbol{\beta}`
```

Comment:
Why call them `params` instead of `beta` and `gamma` separately?

Reply:
Agreed. Please update to ``def(gammas, betas, ...)``